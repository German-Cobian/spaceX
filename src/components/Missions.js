import React from 'react';

const Missions = () => {
  return (
    <div>
      <h3>This is Missions Component</h3>
    </div>
  )
};

export default Missions;
