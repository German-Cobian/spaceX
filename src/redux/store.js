import { createStore, combineReducers, applyMiddleware } from 'redux';
import thunk from 'redux-thunk';
import logger from 'redux-logger';
import missionsReducer from './missionsReducer';

const reducer = combineReducers({
  missionsReducer,
  // additional reducers could be added here
});

const store = createStore(
  reducer,
  applyMiddleware(logger, thunk),
);

export default store;
