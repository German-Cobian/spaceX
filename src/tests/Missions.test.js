import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Missions from '../components/Missions';
import store from '../redux/store';

test('matches RocketsProfile component snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Missions />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
