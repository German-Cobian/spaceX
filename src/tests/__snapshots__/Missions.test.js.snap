// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches RocketsProfile component snapshot 1`] = `
<div
  className="col-11 mx-auto mt-4"
>
  <table
    className="table table-sm table-striped table-bordered table-hover"
    style={
      Object {
        "textAlign": "left",
      }
    }
  >
    <thead>
      <tr>
        <th
          className="col-1"
        >
          Missions
        </th>
        <th
          className="col-8"
        >
          Description
        </th>
        <th
          className="col-1"
        >
          Status
        </th>
        <th>
           
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
